plugins/CoreHome/tests/UI/WelcomeWidget_spec.js

```diff
@@ -0,0 +1,37 @@
+/*!
+ * Piwik - free/libre analytics platform
+ *
+ * Dashboard screenshot tests.
+ *
+ * @link http://piwik.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+
+describe('WelcomeWidget', function () {
+    var url = "";
+
+    it('should load correctly', async function () {
+        await page.goto(url);
+        await page.waitForNetworkIdle();
+        expect (await page.screenshotSelector('#widgetCoreHomegetPromoVideo')).to.matchImage('load');
+    });
```

Comment:
Is this widget included in one of the dashboard tests? If so, we may not need to add a separate UI test for a widget like this. Given how heavy the UI tests are, it's better to avoid adding to them when possible IMO.

I'm not sure about the other test as well, if we're just testing the links we could do so in a PHP test. @tsteur what are your thoughts?

Reply:
Not needed in this case. If we can avoid a UI test that is always welcome :)

Reply:
OK then, tests have been removed.